navigating-many-to-one.mdx

```diff
@@ -0,0 +1,335 @@
+---
+title: 'Managing many-to-one and one-to-many database relations'
+description: 'Addressing the N+1 problem and navigating one-to-many relationships with Xata's new approach.'
+image:
+  src: 'https://raw.githubusercontent.com/xataio/mdx-blog/main/images/csv-import-export-cover.jpg'
+  alt: 'Many-to-one relationship'
+author: Alejandro Martínez Vieites
+date: 09-01-2023
+published: true
+slug: navigating-many-to-one
```

Comment:
When we have the image, let's drop in an `ogImage` attribute here to use Elizabets design. Looks like we have one here: https://github.com/xataio/mdx-blog/issues/23#issuecomment-1701330014